Jenkinsfile

```diff
@@ -10,84 +10,62 @@
  * Copyright Contributors to the Zowe Project.
  */
 
-
-BUILD_SERVER_TMP_DIR = "/ZOWE/tmp"
-JAVA_HOME = "/usr/lpp/java/J8.0_64"
-ANT_HOME = "/ZOWE/apache-ant-1.10.5"
-XLC_DATASET = "CBC.SCCNCMP"
-
-
 node("ibm-jenkins-slave-nvm") {
 
     def lib = library("jenkins-library").org.zowe.jenkins_shared_library
     def pipeline = lib.pipelines.generic.GenericPipeline.new(this)
 
-    def buildDir = "${BUILD_SERVER_TMP_DIR}/~${env.BUILD_TAG}"
-    def buildServer = null
-
-    withCredentials([usernamePassword(
-        credentialsId: lib.Constants.DEFAULT_PAX_PACKAGING_SSH_CREDENTIAL,
-        usernameVariable: "BUILD_SERVER_USERNAME",
-        passwordVariable: "BUILD_SERVER_PASSWORD"
-    )]) {
-        buildServer = [
-            name         : lib.Constants.DEFAULT_PAX_PACKAGING_SSH_HOST,
-            host         : lib.Constants.DEFAULT_PAX_PACKAGING_SSH_HOST,
-            port         : lib.Constants.DEFAULT_PAX_PACKAGING_SSH_PORT.toInteger(),
-            user         : BUILD_SERVER_USERNAME,
-            password     : BUILD_SERVER_PASSWORD,
-            allowAnyHosts: true
-        ]
-    }
-
     pipeline.admins.add("dnikolaev", "sgrady")
 
-    pipeline.setup()
-
-    pipeline.createStage(
-        name: 'Deploy source',
-        stage: {
-            sh "tar --exclude='./source.tar' -cf source.tar *"
-            sshCommand remote: buildServer, command: "rm -rf ${buildDir} && mkdir -p \$_"
-            sshPut remote: buildServer, from: "source.tar", into: buildDir
-            sshCommand remote: buildServer, command: \
-                """
-                cd ${buildDir} &&
-                pax -r -x tar -o to=IBM-1047 -f source.tar
-                """
+    pipeline.setup(
+        packageName: 'org.zowe.zss',
+        github: [
+            email                      : lib.Constants.DEFAULT_GITHUB_ROBOT_EMAIL,
```

Comment:
Do we have to pass all the defaults explicitly?
I was under impression that the defaults are applied automatically, aren't they?

Reply:
Unfortunately those are not applied automatically for now. I was hesitating to do so,  but probably it’s better to do.

Reply:
Will be added with https://github.com/zowe/jenkins-library/issues/41